tests/wpt/mozilla/tests/css/float_relative_to_position.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html>
+<head>
+<meta charset=utf-8>
+<title>Tests the relationship between float and position</title>
+<head>
+<body>
+<div style="position: fixed; float: left"></div>
+
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script>
+    test(function() {
+        var computed_style = getComputedStyle(document.querySelector("div"))["float"];
+        assert_equals(computed_style, "none");
+
+        document.querySelector("div").setAttribute("position", "absolute");
```

Comment:
The `position` attribute on `<div>` has no meaning.  Did you mean to set the `style` attribute?

The fact that you could make this mistake and have the test still pass indicates that it's not ideal... maybe add a stylesheet with a style like `div { float: left }`, have the test check that float is left, then change the `position` using the style attribute.